Formula/p11-kit.rb

```diff
@@ -0,0 +1,27 @@
+class P11Kit < Formula
+  desc "Library to load and enumerate PKCS#11 modules"
+  homepage "https://p11-glue.freedesktop.org"
+  url "https://p11-glue.freedesktop.org/releases/p11-kit-0.23.2.tar.gz"
+  sha256 "ba726ea8303c97467a33fca50ee79b7b35212964be808ecf9b145e9042fdfaf0"
+
+  depends_on "pkg-config" => :build
+  depends_on "libffi"
+  depends_on "libtasn1"
+
+  def install
+    # https://bugs.freedesktop.org/show_bug.cgi?id=91602#c1
+    ENV["FAKED_MODE"] = "1"
+
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--disable-trust-module",
+                          "--prefix=#{prefix}"
```

Comment:
I would suggest adding `--sysconfdir=#{etc}`.  Also, if you don't add `--with-module-config=#{etc}/pkcs11/modules`, then any modules that are created end up going into the `#{prefix}` path - which will change every time p11-kit is updated.